gammapy/modeling/parameter.py

```diff
@@ -220,6 +222,23 @@ def quantity(self, val):
         self.value = val.value
         self.unit = val.unit
 
+    def _check_value_within_bounds(self):
+        """Send a warning if value is outside the min/max range (or at the border within 1%)"""
+        min_nan = np.isnan(self.min)
+        max_nan = np.isnan(self.max)
+        min_ok = self.value > self.min and ~np.isclose(
+            self.value - self.min, 0, atol=1e-2
```

Comment:
The docstring states it would be a relative tolerance but `np.isclose()` uses `atol` here. In any case the 1% seems a like an arbitrary choice to me. I checked and e.g. [sherpa emits a warning](https://github.com/sherpa/sherpa/blob/358ab71a0e1f1e639a33e71bcbd3b4fa0fe7a979/sherpa/fit.py#L420) only in case of equality of the value to the boundary. Have you tested what happens without the tolerance?

Reply:
Addendum: for `iminuit` it might be that without the tolerance this warning is maybe never emitted, because the parameter values only approach the boundary (because of the `atanh` parameter scaling...). Does `iminuit ` emits a warning on its own?

Reply:
> The docstring states it would be a relative tolerance but np.isclose() uses atol here

Ups! I fix it sorry

> for iminuit it might be that without the tolerance this warning is maybe never emitted, because the parameter values only approach the boundary (because of the atanh parameter scaling...)

Exactly, I think a standard equality will never be satisfied for a MINUIT fit. That's why I introduced this tolerance value.

> Does iminuit emit a warning on its own?

I don't think so, otherwise we wouldn't have this issue at all, no?